spring-boot-starter-mesos/src/main/java/com/containersolutions/mesos/scheduler/config/MesosConfigProperties.java

```diff
@@ -13,6 +14,7 @@
     private ResourcesConfigProperties resources = new ResourcesConfigProperties();
     private String principal;
     private String secret;
+    private List<String> uri;
```

Comment:
If you instantiate the list here (`new ArrayList<>()`), you don't have to parse it through `Optional` above.